check_pgactivity

```diff
@@ -6372,6 +6376,74 @@ sub check_wal_files {
     return ok( $me, \@msg, \@perfdata );
 }
 
+=item B<stat_snapshot_age> (8.1+)
+
+Check the age of the statistics snapshot (statistics collector's statistics).
+This probe help to detect a frozen stats collector process.
+
+Perfdata returns the statistics snapshot age.
+
+Critical and Warning thresholds accept a raw number of seconds.
+
+=cut
+
+sub check_stat_snapshot_age {
+    my $me       = 'POSTGRES_STAT_SNAPSHOT_AGE';
+    my @rs;
+    my @perfdata;
+    my @msg;
+    my @hosts;
+    my %args     = %{ $_[0] };
+    my $stats_age;
+
+    my %queries  = (
+    $PG_VERSION_95 => q{ SELECT extract(epoch from (pg_stat_get_snapshot_timestamp() - now())) AS age }
+    );
```

Comment:
why using `%queries` and `query_ver` when you have only one query for 9.5+? ISTM a simple query and using the `query()` sub would be enough.